General/ExportFormats.md

```diff
@@ -0,0 +1,138 @@
+
+In order to be able to save some code in readable files, Pharo includes some textual export formats. This page will explain the main export formats used to save code in VCS such as git.
+
+This page will present formats from the most recommanded for new projects to the less recommanded.
+
+- [Comparisons](#comparisons)
+  * [Overview](#overview)
+  * [Supported Pharo versions](#supported-pharo-versions)
+- [Tonel](#tonel)
+  * [Tonel Pros and Cons](#tonel-pros-and-cons)
+  * [Tonel supported Pharo versions](#tonel-supported-pharo-versions)
+- [FileTree metadata less](#filetree-metadata-less)
+- [FileTree metadata less Pros and Cons](#filetree-metadata-less-pros-and-cons)
+  * [FileTree metadata less supported Pharo versions](#filetree-metadata-less-supported-pharo-versions)
+- [FileTree metadata full](#filetree-metadata-full)
+  * [FileTree metadata full Pros and Cons](#filetree-metadata-full-pros-and-cons)
+  * [FileTree metadata full supported Pharo versions](#filetree-metadata-full-supported-pharo-versions)
+
+
+## Comparisons
+
+### Overview 
+
+|| Tonel | FileTree metadata less | FileTree metadata full |
+| ------------- | ------------- |------------- |------------- |
+| Work out of the box since Pharo | 6.1 | 4 | 3 |
+| Export granularity | Class | Method | Method |
+| Works well on windows | :white_check_mark: | :x: | :x: |
+| Easy to merge two branches | :white_check_mark: | :white_check_mark: | :x: |
+| Takes a reasonable space on file system | :white_check_mark: | :x: | :x: |
+| Works with Iceberg | :white_check_mark: | :white_check_mark: | :x: |
+| Works with Monticello | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+
+### Supported Pharo versions
+
+| Pharo Version | FileTree metadata full | FileTree metadata less | Tonel |
+| ------------- | ------------- |------------- |------------- |
+3 |  :white_check_mark: | Update Metacello? | :x: |
+4 |  :white_check_mark: |  :white_check_mark: | Update Metacello + Install Tonel |
+5 |  :white_check_mark: |  :white_check_mark: | Update Metacello + Install Tonel |
+6.0 |  :white_check_mark: |  :white_check_mark: | Update Metacello + Install Tonel |
+6.1 | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+7 | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+8 | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+
+## Tonel 
+
+Tonel ([https://github.com/pharo-vcs/tonel](https://github.com/pharo-vcs/tonel)) is an export format introduced in Pharo 6.1. 
+
+It creates one file by class or extented class and works well on windows (compared to file tree).
+
+### Tonel Pros and Cons
+
+Pros
+- Works well on windows 
+- Not much space lost because of small files
+- Export readable files
+- No problem during merge because of metadata
+
+Cons
+- Work out of the box only since Pharo 6.1
```

Comment:
```suggestion
- Works out of the box since Pharo 6.1, not officially supported by older versions.
```

Reply:
I do not agree. Tonel is officially supported (The checks are done for Pharo 5 to 7 and we try to keep p4 compatibility too). It's just that it is not out of the box and requires to be installed.

Reply:
If you agree, can you edit your suggestion to only keep the first part of the sentence please?